dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/DefaultTrackedEntityService.java

```diff
@@ -227,6 +259,29 @@ public TrackedEntity getTrackedEntity(
     return getTrackedEntity(trackedEntityUid, program, params, getCurrentUserDetails());
   }
 
+  private TrackedEntity getNewTrackedEntity(UID uid, Program program, TrackedEntityParams params)
```

Comment:
Is this really returning a detached TrackedEntity? If so, I feel we should rename the "getNewTrackedEntity" into something else, maybe even "getDetachedTrackedEntity"... or something more clearer.

Reply:
`getNewTrackedEntity` is a temporary name. All tracker exporters should return detached entities. `getNewTrackedEntity` is slowly replacing `getTrackedEntity` at which point it will itself be called `getTrackedEntity`